### ISSUE TITLE
libximc fails to install unless libxibridge is provided

### ISSUE THREAD

**Issue by petrmanek** [EPC-MSU/libximc#11]
I think I discovered a bug.

When I configure libximc v2.15.7 as `--without-xibridge`, make and install the project, I get the following error:

```
Making install in libximc
make[1]: Entering directory '/tmp/libximc/libximc'
Making install in src
make[2]: Entering directory '/tmp/libximc/libximc/src'
make  install-am
make[3]: Entering directory '/tmp/libximc/libximc/src'
make[4]: Entering directory '/tmp/libximc/libximc/src'
mkdir -p "/tmp/prefix/lib"
cp -Ra /libxibridge.so "/tmp/prefix/lib"
cp: cannot stat '/libxibridge.so': No such file or directory
```

It appears that [this line](https://github.com/EPC-MSU/libximc/blob/3496c6e103091d40defa85523305d9b163330992/libximc/src/Makefile.am#L19) in a autoconf'd `Makefile` does not care about `HAVE_XIBRIDGE`. An easy fix would be to surround it with an `if` block.

**Comment by shampletov-no** [comment]
You are right! That's a bug.  And an easy if-block really works.

Investigating that problem, I also noticed several new bugs like unused-variable errors while using no xibridge. So it's also to be fixed. Working on it

**Comment by shampletov-no** [comment]
The bug was fixed in the following commits: [commit#1](https://github.com/EPC-MSU/libximc/commit/9da8e2f0f494b476e23114641e074a78b64cf565) ,[commit#2](https://github.com/EPC-MSU/libximc/commit/d2b7513ea763ef842b16051a6568d7674122dc2f), [commit#3](https://github.com/EPC-MSU/libximc/commit/bcab527b7251a2a9f50b53d09fc5b88814718b83). You can build with `--without-xibridge` option from the last commit.

In case of any other errors, we will appreciate you let us know.

**Comment by petrmanek** [comment]
Splendid, I will try it and let you know if I find something wrong. Thank you for prompt resolution!
